### PR TITLE
fix(CX-2995): Center MyC empty states

### PR DESCRIPTION
**Pull request by MrSltun**
### Description

This PR resolves [CX-2995]

This PR updates MyC Insights empty state and fixes the centering issue with My Collection empty states

### Screenshots

| Android | iOS | 
| -- | -- |
| image | image | 
| image | image |
| image | image |

#### Acceptance Criteria

-

#### Setup Instructions

-

### PR Checklist

- [X] I tested my changes on **iOS** / **Android**.
- [X] I added screenshots or videos to illustrate my changes.
- [x] I added Tests and Stories for my changes.
- [ ] I added an [app state migration].
- [ ] I hid my changes behind a [feature flag].
- [ ] I have prefixed changes that need to be tested during a release QA with **[NEEDS EXTERNAL QA]** on the changelog.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to run this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

[CX-2995]: https://artsyproduct.atlassian.net/browse/CX-2995?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by olerichter00 — APPROVED** [review]
⚖️......⚖️......⚖️.......⚖️
.
.
..............🍏............... 
.
.
⚖️......⚖️......⚖️.......⚖️

File: src/app/Components/StickyTabPage/StickyTabPageFlatList.tsx [hunk 1]
```diff
@@ -130,10 +130,7 @@ export const StickyTabPageFlatList: React.FC<StickyTabFlatListProps> = (props) =
         ListHeaderComponent={
           <Animated.View
             onLayout={() => setHeaderDidMount(true)}
-            style={{
-              flex: 1,
-              height: staticHeaderHeight,
```
Comment (dariakoko) [inline src/app/Components/StickyTabPage/StickyTabPageFlatList.tsx:135]: Ouch 🙈 that doesn't look right
Good catch

File: src/app/Scenes/MyCollection/MyCollectionArtworks.tsx [hunk 2]
```diff
@@ -203,7 +203,7 @@ const MyCollectionZeroState: React.FC = () => {
   return (
     <ZeroState
       bigTitle="Your Art Collection in Your Pocket"
-      subtitle="Access market insights and manage  your collection online."
+      subtitle="Access market insights and manage your collection online. "
```
Comment (dariakoko) [inline src/app/Scenes/MyCollection/MyCollectionArtworks.tsx:206]: Extra space 👀 Let's remove it?

**Review by dariakoko — APPROVED** [review]
Looks good!
I left only one tiny little comment

File: src/app/Scenes/MyCollection/MyCollectionArtworks.tsx (hunk 2 above)
Comment (dariakoko) [reply src/app/Scenes/MyCollection/MyCollectionArtworks.tsx:206]: And fix this string in tests as well

**Comment by dariakoko** [comment]
> ⚖️......⚖️......⚖️.......⚖️ . . ..............🍏............... . . ⚖️......⚖️......⚖️.......⚖️

One dot on the left side of the apple is missing! 😱

File: src/app/Scenes/MyCollection/MyCollectionArtworks.tsx (hunk 2 above)
Comment (MrSltun) [reply src/app/Scenes/MyCollection/MyCollectionArtworks.tsx:206]: good catch haha

**Review by MounirDhahri — APPROVED** [review]
Such a nice breakdown in the PR description. Thanks for being thoughtful of all details Sultan

**Pull request merged**